Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -495,6 +496,48 @@ Messages::WebDriverClient::GetWindowHandlesResponse WebDriverConnection::get_win
     return handles;
 }
 
+// 11.5 New Window, https://w3c.github.io/webdriver/#dfn-new-window
+Messages::WebDriverClient::NewWindowResponse WebDriverConnection::new_window(JsonValue const& payload)
+{
+    // 2. If the current top-level browsing context is no longer open, return error with error code no such window.
+    TRY(ensure_open_top_level_browsing_context());
+
+    // 3. Handle any user prompts and return its value if it is an error.
+    TRY(handle_any_user_prompts());
+
+    // 4. Let type hint be the result of getting the property "type" from the parameters argument.
+    auto type_hint = TRY(get_property(payload, "type"sv));
+
+    // 5. Create a new top-level browsing context by running the window open steps with url set to "about:blank",
+    //    target set to the empty string, and features set to "noopener" and the user agent configured to create a new
+    //    browsing context. This must be done without invoking the focusing steps for the created browsing context. If
+    //    type hint has the value "tab", and the implementation supports multiple browsing context in the same OS
+    //    window, the new browsing context should share an OS window with the current browsing context. If type hint
+    //    is "window", and the implementation supports multiple browsing contexts in separate OS windows, the
+    //    created browsing context should be in a new OS window. In all other cases the details of how the browsing
+    //    context is presented to the user are implementation defined.
+    auto browsing_context = this->m_page_client.new_client_from_current();
+
+    // 6. Let handle be the associated window handle of the newly created window.
+    auto handle = Core::DateTime::now().to_string();
+
+    // 7. Let type be "tab" if the newly created window shares an OS-level window with the current browsing context,
+    //    or "window" otherwise.
+    auto type = "tab"sv.to_string();
```

Comment:
Don't need `.to_string()`
```suggestion
    auto type = "tab"sv;
```